api/src/main/java/com/khali/api3/domain/user/User.java

```diff
@@ -44,9 +44,6 @@ public class User {
     @Column
     private String password;
 
-    @Column
-    private Boolean active;
-
     @Column(name = "insert_date")
```

Comment:
falta adicionar `@Transient` ao `insert_date` de `User`
```suggestion
    @Transient
    @Column(name = "insert_date")
```
favor verificar também as outras classes para a inserção do `@Transient`

Reply:
esse ajuste já foi resolvido no branch `user_controller`